paasta_tools/marathon_tools.py

```diff
@@ -689,19 +677,28 @@ class InvalidMarathonHealthcheckMode(Exception):
 
 class MarathonDeployStatus:
     """ An enum to represent marathon app deploy status.
-    'Not Running' is NOT included to keep consistent with mesosphere documentation.
     Changing name of the keys will affect both the paasta CLI and API.
     """
-    Running, Deploying, Stopped, Delayed, Waiting = range(0, 5)
+    Running, Deploying, Stopped, Delayed, Waiting, NotRunning = range(0, 6)
 
     @classmethod
     def tostring(cls, val):
         for k, v in vars(cls).iteritems():
             if v == val:
                 return k
 
+    @classmethod
+    def fromstring(cls, str):
+        return getattr(cls, str, None)
+
+
+def get_marathon_app_deploy_status(app_id, client):
```

Comment:
it seems odd to me that this returns 2 different types according to the running state of the app (an app, or a tuple of (``MarathonDeployStatus`` and an int). Given the function is called ``get_marathon_app_deploy_status``, I'd expect it to always return a value of ``MarathonDeployStatus``. Please also just make this return the DeployStatus, and make the caller decide if they want the delay, too.